src/Voron/Data/Tables/Table.cs

```diff
@@ -415,7 +415,7 @@ public long Insert(TableValueBuilder builder)
                 page.Flags = PageFlags.Overflow | PageFlags.RawData;
                 page.OverflowSize = size;
 
-                ((RawDataOverflowPageHeader*)page.Pointer)->SectionOwnerHash = Hashing.XXHash64.Calculate(Name.Content.Ptr, (ulong)Name.Content.Length);
+                ((RawDataOverflowPageHeader*)page.Pointer)->SectionOwnerHash = ActiveDataSmallSection.SectionOwnerHash;
                 ((RawDataOverflowPageHeader*)page.Pointer)->TableType = _tableType;
```

Comment:
Can we use the same trick for the `TableType` and then not have to pass it except during creation?